samples/manifests/plugins/dev-env-with-plugins.yaml

```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```

Comment:
@rb201 - Please review this change. I was able to deploy by removing cognito attributes from manifest file.

Reply:
Added cognito attributes to manifest files.